service: add a functoin to help create fuse vfs backend

taoohong opened the pull request:
Add a function to get fuse backend vfs,
reduce explicit references to fuse_backend_rs.

jiangliu commented on service/src/fusedev.rs:
```diff
@@ -614,3 +614,33 @@ pub fn create_fuse_daemon(
 }
+
+/// Prepare the backend vfs when using rafs or passthrough as the fuse filesystem driver
+pub fn get_backend_vfs(
```
create_vfs_backend()?

taoohong replied:
done

taoohong committed:
service: add a functoin to help create fuse vfs backend

Add a function to help create fuse vfs backend,
reduce explicit references to crate fuse_backend_rs.

Signed-off-by: taohong <taoohong@linux.alibaba.com>
